Guides/TLS-Guide.md

```diff
@@ -79,3 +79,35 @@ let client = try MongoClient(
 ```
 **Note**: In both cases, if both a client certificate and a client private key are needed, the files should be concatenated into a single file which is specified by `tlsCertificateKeyFile`.
 
+## Server Certificate Validation
```

Comment:
this was mostly taken from the [C driver's docs on TLS/OCSP](http://mongoc.org/libmongoc/current/configuring_tls.html#server-certificate-verification).